static/src/javascripts/projects/common/modules/identity/api.js

```diff
@@ -0,0 +1,284 @@
+// @flow
+
+/* global escape:true */
+import { ajax } from 'lib/ajax';
+import config from 'lib/config';
+import { getCookie as getCookieByName } from 'lib/cookies';
+import mediator from 'lib/mediator';
+import { local } from 'lib/storage';
+import { mergeCalls } from 'common/modules/asyncCallMerger';
+
+/**
+ * Left this as an object as there are only static methods
+ * We'll need to change this once there is some state change
+ * TODO(jamesgorrie): Allow this to show policies too (not needed yet)
+ */
```

Comment:
Let's get rid of this comment as it no longer applies

Reply:
Fixed in e821cb41c11e8aaa81c03cd5239ef5f8f2c5846b